src/components/Images/Images.md

```diff
@@ -1,5 +1,14 @@
 ```jsx
 <div style={{ width: 650 }}>
+  <CloudinaryImage
+    crop="fill"
+    alt="father and kids playing"
+    width={[250,350,450,550]}
```

Comment:
re question about boolean. If that IS a thing, can you   show an example here or put a comment above the jsx providing for how that works?